File: src/main/java/com/ddubson/models/Author.java
package com.ddubson.models;

import lombok.Builder;
import lombok.Data;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name="authors")
@Data
@Builder
public class Author {
    @Id
    @GeneratedValue
    private Long id;

    private String firstName;
    private String lastName;
}

File: src/main/kotlin/com/ddubson/models/Author.kt
package com.ddubson.models

import java.io.Serializable
import javax.persistence.*

@Entity
@Table(name = "authors")
data class Author(@Id @GeneratedValue val id: Long,
                  val firstName: String,
                  val lastName: String,
                  @OneToMany(mappedBy = "author") val books: List<Book>) : Serializable